deploy/charts/harvester/dependency_charts/harvester-network-controller/templates/daemonset.yaml

```diff
@@ -21,21 +21,30 @@ spec:
       serviceAccountName: {{ include "harvester-network-controller.name" . }}
       hostNetwork: true
       containers:
-        - name: network
-          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
-          imagePullPolicy: {{ .Values.image.pullPolicy }}
-          command:
-            - harvester-network-controller
-          args:
-            - agent
-          securityContext:
-            privileged: true
-          volumeMounts:
-          - mountPath: /dev
-            name: dev
-          - mountPath: /lib/modules
-            name: modules
-          resources:
+      - name: network
+        image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+        imagePullPolicy: {{ .Values.image.pullPolicy }}
+        command:
+        - harvester-network-controller
+        args:
+        - agent
+        env:
+        - name: NODENAME
+          valueFrom:
+            fieldRef:
+              apiVersion: v1
+              fieldPath: spec.nodeName
+        securityContext:
+          privileged: true
+          capabilities:
+            add:
+            - NET_ADMIN
```

Comment:
The privileged gives full capabilities. Why do we need both set?

Reply:
It can be deleted according to my test.